store/src/main/java/com/denis/store/utility/ConnectionPool.java

```diff
@@ -0,0 +1,24 @@
+package com.denis.store.utility;
+
+import org.apache.commons.dbcp2.BasicDataSource;
+
+import java.sql.Connection;
+import java.sql.SQLException;
+
+public class ConnectionPool {
+
+    private static BasicDataSource ds = new BasicDataSource();
+
+    static {
+        ds.setUrl("jdbc:h2:~/store");
```

Comment:
please move this settings to properties file, on any other external configuration, not hardcode creds and urls in code